emit-ch4plume-v1/index.html

```diff
@@ -17,6 +17,8 @@
   <body>
     <div id='map'></div>
     <div class="marker"></div>
+    <div id="lds-roller-id" class="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
```

Comment:
lol what's going on here

Reply:
We need multiple divs for the spinner to work 🤷

Reply:
whaaat howww

Reply:
Because in CSS we are moving the divs
```
.lds-roller div {
  animation: lds-roller 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
  transform-origin: 40px 40px;
}
.lds-roller div:after {
  content: " ";
  display: block;
  position: absolute;
  width: 7px;
  height: 7px;
  border-radius: 50%;
  background: #fff;
  margin: -4px 0 0 -4px;
}
.lds-roller div:nth-child(1) {
  animation-delay: -0.036s;
}
```

Reply:
thats one way to do it

Reply:
This should probably change in the future to be consistent  with VEDA "waiting spinner".